content/en/blog/_posts/2022-11-27-Vulnerable-Microservices/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: "Vulnerable Microservices"
+date: 2022-11-27
+slug: vulnerable-microservices
+---
+
+
+**Author:** 
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy relying on constructing a non-vulnerable, weakness-free service in 202x is clearly too naïve. It seems the only viable strategy is to:
+
+***
+<h3 align="center">
+Admit your services are vulnerable
+</h3>
+
+***
```

Comment:
```suggestion
_Admit that your services may be vulnerable._
```

Only use HTML if there's a really good case for it. This doesn't feel like one.

Reply:
I did nit find a new way to get this test standout as it shoyld. Unless tgere are other thoughts, I will revert back to the original HTML.

Reply:
Please see https://github.com/kubernetes/website/pull/38104#pullrequestreview-1202795661